contracts/Liquidator/Liquidator.sol

```diff
@@ -350,4 +386,50 @@ contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable {
             revert TreasuryPercentTooHigh(maxTreasuryPercentMantissa, treasuryPercentMantissa_);
         }
     }
+
+    function _checkForceVAILiquidate(address vToken, address borrower) private view {
+        uint256 vaiDebt_ = vaiController.getVAIRepayAmount(borrower);
+        if (!forceVAILiquidate || vaiDebt_ * 10 ** 18 < minLiquidatableVAI || vToken == address(vaiController)) return;
```

Comment:
I think `vaiController.getVAIRepayAmount` returns the mantissa value of the VAI debt. Am I wrong? In that case, I think we don't need to multiply by 10**18

Reply:
`vaiController.getVAIRepayAmount` returns the quantity of VAI that have been minted for a borower, and as VAI contains 18 decimals, you are correct that there is no need to multiply by 10**18.

Reply:
Would it make sense to do a fork test covering this change?